app/views/comments/_comment.html.erb

```diff
@@ -5,7 +5,14 @@
     <br>
     <p>
       <strong>User:</strong>
-      <%= comment.user.name %>
+      <% if comment.anonymous && !(current_user.admin? || current_user == comment.user) %>
+        Anonymous

```

Comment:
Is the author not able to see their own name under an anonymous comment that they made?

Reply:
The author will be given the same thing that admin will see. They will see their name and a message next to their name that it was posted anonymously. I will add comments in the code to explain it better.